### PR TITLE
New first aid kits

### PR DESCRIPTION
**Pull request by Peptide90**
## About the PR 
This adds a bunch of first aid kits based on TG but in a few cases customised to be more representative of the contents.
I created sprites for the burn, toxins, o2 and radiation kits and tried to make inhands sprites where possible for them plus for combat and purple kits.

See screenshot for kits but for list:
- Black Kit - Combat medical kit (can be used to easily create a fill for syndicate meds (e.g. injectors and advanced stuff).
- White Kit - Generic first aid kit
- White + Red kit - Advanced first aid kit (has advanced healing items for quick healing).
- Yellow kit - Burn treatment kit
- Pink kit - Brute trauma treatment kit
- Red kit - Radiation treatment kit
- Blue kit - Oxygen deprevation treatment kit
- Green kit - Toxin treatment kit

Corrected fills filepath from entities / specific to catalog and created basic fills for regular and burn kit. Remaining kits don't have relevant items but left comments on where to find the ideal loadouts for them. Also signposted with comments where to find the resources and entities so it can be easily modified in future.

Also removed old icon and added gauze as a medical item.

**Screenshots**
firstaidkits

Icons total 11.2 KB

:cl:
- add: Added new first aid kits with fancy new sprites.

### CODE REVIEW

**Comment by PJB3005** [comment]
The O on the O2 kit sprite feels a little blurry and washed out, as if due to bad AA.

**Comment by Visne** [comment]
> 
> 
> The O on the O2 kit sprite feels a little blurry and washed out, as if due to bad AA.

Same goes for burn, toxin and rad kit.

**Comment by tmtmtl30** [comment]
I think the rad one is absolutely fine, the toxin one is similar. The burn one sort of has that problem, but I'm fine with it... the O2 one I can see the case against, but I can also see why it might be hard to make it look decent while keeping it better defined. All in all, I'd say they're fine.

**Comment by PJB3005** [comment]
I just realized that you'll probably have to change the red cross on one of those sprites to... not be a red cross. Video games aren't allowed to use the symbol (violates Geneva Convention) and from time to time this does get enforced. Better not to take the risk.

**Comment by Visne** [comment]
SS14 almost commiting war crimes 😳

**Comment by UnderscoreX5** [comment]
Quite dumb but actually true, but what could we have instead of the cross?

**Comment by Peptide90** [comment]
> 
> 
> Quite dumb but actually true, but what could we have instead of the cross?

White cross. Just can't be red I guess.

**Comment by UnderscoreX5** [comment]
What about an orange cross? Most people wouldnt notice that much of a difference

**Comment by PJB3005** [comment]
> What about an orange cross? Most people wouldnt notice that much of a difference

Wikipedia actually lists that US ambulances *used* to use an orange cross but it was decided that was too close to red and was changed. So...

**Comment by Peptide90** [comment]
> 
> 
> > What about an orange cross? Most people wouldnt notice that much of a difference
> 
> Wikipedia actually lists that US ambulances _used_ to use an orange cross but it was decided that was too close to red and was changed. So...

Does dark blue and black tickle your fancy? For the advkit. 
advkit

**Comment by UnderscoreX5** [comment]
Not gonna lie, dark blue looks awesome!

File: Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		"x": 32,
+		"y": 32
+	},
+	"license": "Taken from https://github.com/tgstation/tgstation/tree/727eb0a445bccbdc2d472e158e96b87fc0e997a1. Rad, toxin, o2, fire and adv by peptide",
+	"copyright": "CC-BY-SA-3.0",
+	"states": [
+		{
+			"name": "firstaid"
+		},
+		{
```
Comment (SweptWasTaken) [inline Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/meta.json:13]: !!!

File: Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            "directions": 4
+        },
+        {
+            "name": "brutekit-inhand-left",
+            "directions": 4
+        },
+		{
+			"name": "burnkit"
+		},
+		{
```
Comment (SweptWasTaken) [inline Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/meta.json:35]: !!!

File: Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            "directions": 4
+        },
+        {
+            "name": "burnkit-inhand-left",
+            "directions": 4
+        },
+		{
+			"name": "o2kit"
+		},
+		{
```
Comment (SweptWasTaken) [inline Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/meta.json:46]: !!!

**Review by SweptWasTaken — APPROVED** [review]

**Comment by SweptWasTaken** [comment]
Fix your RSI meta.json and conflicts and this should be g2g. I like the sprites.

**Comment by Peptide90** [comment]
> 
> 
> Fix your RSI meta.json and conflicts and this should be g2g. I like the sprites.

All fixed and good to go

**Pull request merged**